themes/bootstrap5/scss/variables.scss

```diff
@@ -16,3 +16,20 @@ $nav-link-hover-bg:       $gray-lighter !default;
 
 $alert-default-bg: #f5f5f5 !default;
 $alert-default-border-color: darken($alert-default-bg, 7%) !default;
+
```

Comment:
These styles in variables.scss replace the global (not color-scheme specific) declarations in bootstrap-variable-overrides.scss.